test-no-std/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "test-no-std"
-version = "0.7.0"
+version = "0.8.0"
```

Comment:
I don't think there've been breaking changes. Only a feature has been added and some crates. No break, or was there?